subsys/usb/host/usbh_ch9.c

```diff
@@ -170,3 +170,33 @@ int usbh_req_clear_sfs_rwup(const struct device *dev,
 			      bmRequestType, bRequest, wValue, 0, 0,
 			      NULL);
 }
+
```

Comment:
Please change commit title to `usb: host: add port power and port reset USB hub features`